mautic/mautic
# [UI] Toolbar batch actions

| Q                                      | A
| -------------------------------------- | ---
| Bug fix? (use the a.b branch)          | 🔴 
| New feature/enhancement? (use the a.x branch)      | 🟢
| Deprecations?                          | 🔴
| BC breaks? (use the c.x branch)        | 🔴
| Automated tests included?              | 🔴 
| Related user documentation PR URL      | mautic/user-documentation#... 
| Related developer documentation PR URL | mautic/developer-documentation-new#... 
| Issue(s) addressed                     | Fixes #13155 



## Description

This PR moves list bulk actions to the toolbar.

https://github.com/user-attachments/assets/21dcf457-97c7-4bf7-9d19-73ee1d5c1b81

Also:
- adds a counter for selected items
- makes export and delete buttons always visible
- creates a new location in php for the buttons that are always visible in the batch actions toolbar
- adds a LESS file for the toolbar styles
- wipe some CSS workarounds used for the old structure in page lists
- maintains compatibility with the tableheader template, so anything using it will keep working

Dependency:
- #14276 






---
### 📋 Steps to test this PR:


1. Open this PR on Gitpod or pull down for testing locally (see docs on testing PRs [here](https://contribute.mautic.org/contributing-to-mautic/tester))
2. Open Contacts
3. Create a contact
4. Create another
5. Back to the list
6. Click on the checkbox from the header to select all
7. See the toolbar
8. Repeat for other pages (companies, segments etc)
